ophyd/controls/mca.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import print_function
+import logging
+
+from collections import namedtuple
+
+from .signal import (EpicsSignal, EpicsSignalRO)
+from .device import OphydDevice
+from .device import (Component as C, DynamicDeviceComponent as DDC)
+
+logger = logging.getLogger(__name__)
+
+
+_roi_field_map = {'name': (EpicsSignal, '.R{n}NM'),
```

Comment:
This needs to be an ordered dict or the named tuple will randomly re-order process-to-process.